tensorflow/python/ops/nn_fused_batchnorm_test.py

```diff
@@ -341,16 +341,16 @@ def testInference(self):
             x_shape, dtype, [1], np.float32, use_gpu=True, data_format='NHWC')
         self._test_inference(
             x_shape, dtype, [1], np.float32, use_gpu=True, data_format='NCHW')
-    self._test_inference(
-        x_shape, np.float32, [1], np.float32, use_gpu=False, data_format='NHWC')
+        self._test_inference(
```

Comment:
This test is removed in the case where no GPU is available, and same for the test below. Like you did for the tests below, swap the "if test.is_gpu_available(cuda_only=True)" and "for dtype in [np.float16, np.float32]" statements

Reply:
Done